src/components/filters/Date/index.tsx

```diff
@@ -94,24 +103,18 @@ function DateFilter({ disabled, header, selectableTableStoreName }: Props) {
                     horizontal: 'left',
                 }}
             >
-                <MenuItem onClick={() => handlers.setFilter('today')}>
-                    <FormattedMessage id="filter.time.today" />
-                </MenuItem>
-                <MenuItem onClick={() => handlers.setFilter('yesterday')}>
-                    <FormattedMessage id="filter.time.yesterday" />
-                </MenuItem>
-                <MenuItem onClick={() => handlers.setFilter('thisWeek')}>
-                    <FormattedMessage id="filter.time.thisWeek" />
-                </MenuItem>
-                <MenuItem onClick={() => handlers.setFilter('lastWeek')}>
-                    <FormattedMessage id="filter.time.lastWeek" />
-                </MenuItem>
-                <MenuItem onClick={() => handlers.setFilter('thisMonth')}>
-                    <FormattedMessage id="filter.time.thisMonth" />
-                </MenuItem>
-                <MenuItem onClick={() => handlers.setFilter('lastMonth')}>
-                    <FormattedMessage id="filter.time.lastMonth" />
-                </MenuItem>
+                <ListItem>
+                    <Typography>
+                        <FormattedMessage id="filter.time.label" />
+                    </Typography>
+                </ListItem>
```

Comment:
Ah, this replaced the tooltip in the intermediate set of screenshots. While the divider helps separate this component (functioning as a header) from the actionable contents of the menu, did you consider increasing the font weight?

Title case is my go-to for menu headers.

Reply:
I can try that. I didn't like the styling but didn't really try any tweaks. I'll up the weight. You thinking like just making it bold?

Reply:
Yes. As far as precedents are concerned, the field selection table settings menu is the only other menu that has a header in the application so I would prefer the general look and feel to be _similar_.

Reply:
I made it `500` 

![image](https://github.com/estuary/ui/assets/270078/55b7a9fa-286a-42a6-a746-95ad91b5dd54)